backend/src/main/java/com/praksa/auction/model/Address.java

```diff
@@ -0,0 +1,91 @@
+package com.praksa.auction.model;
+
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name="address")
+public class Address {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    private String street;
+    private String city;
+    private String zipCode;
+    private String state;
+    private String country;
+
+
+    public Address( String street, String city, String zipCode, String state, String country) {
+        this.street = street;
+        this.city = city;
+        this.zipCode = zipCode;
+        this.state = state;
+        this.country = country;
+    }
+
+    public Address() {
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String getStreet() {
+        return street;
+    }
+
+    public void setStreet(String street) {
+        this.street = street;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public String getZipCode() {
+        return zipCode;
+    }
+
+    public void setZipCode(String zipCode) {
+        this.zipCode = zipCode;
+    }
+
+    public String getState() {
+        return state;
+    }
+
+    public void setState(String state) {
+        this.state = state;
+    }
+
+    public String getCountry() {
+        return country;
+    }
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
We should avoid formatting like this, because it is not good practice and hardly readable.
If something like this is needed in frontend, with less attributes from original entity, dtos can be used.
Second option is to do formatting on frontend itself.
Third option si to have method in backend controller, that will return specific format of address with JsonNode, but if we can avoid, it would be great.